pygameweb/app.py

```diff
@@ -69,11 +70,28 @@ def inject_url_for_image():
         return {'url_for_security': _url_for_security}
 
 
+def http_error_handler(error):
+    message = error.description
+    if '  ' in message:
```

Comment:
What is this bit for?

Reply:
If you `abort(403, 'You are being ratelimited, slow down already')` that second part is the "description" of the error and is useful to pass a custom message for some things

Reply:
Ah, I see. Thanks.